components/server/src/workspace/workspace-starter.ts

```diff
@@ -796,6 +814,11 @@ export class WorkspaceStarter {
             instance.status.message = `Workspace cannot be started: ${err}`;
             await this.workspaceDb.trace({ span }).storeInstance(instance);
             await this.messagebus.notifyOnInstanceUpdate(workspace.ownerId, instance);
+            await this.publisher.publishInstanceUpdate({
+                instanceID: instance.id,
```

Comment:
surprised you still pass the instanceIDher, as I understood it was removed ☝️

Reply:
For Instance Updates, we need to supply the InstanceID, but for Prebuild Updates, we don't pass the instance ID

Reply:
Ah, please ignore the diff was folded.